libraries/lib-theme/AColor.cpp

```diff
@@ -305,6 +305,23 @@ void AColor::Bevel2(wxDC & dc, bool up, const wxRect & r, bool bSel, bool bHighl
       Bmp.GetWidth() - r2, 0, wxCOPY, true );
 }
 
+void AColor::DrawHStretch(wxDC& dc, const wxRect& rect, wxBitmap& bitmap)
+{
+   wxMemoryDC srcDC;
+   srcDC.SelectObject(bitmap);
+
+   const auto sh = bitmap.GetHeight();
+   const auto dh = rect.height;
+   const auto w0 = std::min((bitmap.GetWidth() - 1) / 2, rect.width / 2);
+   const auto dx0 = rect.x;
+   const auto dx1 = rect.x + w0;
+   const auto dx2 = rect.x + rect.width - 1 - w0;
+
+   dc.StretchBlit(dx0, rect.y, w0, dh, &srcDC, 0, 0, w0, sh);
+   dc.StretchBlit(dx1, rect.y, rect.width - w0 * 2, dh, &srcDC, w0, 0, 1, sh);
```

Comment:
Off by one mistake -- the third argument is not `dx1 - dx0`

Reply:
And dx2 - dx1 may be zero or negative, in which case, skip it

Reply:
Right, but `dx2` calculation is wrong

Reply:
Right, but `dx2` calculation is wrong